.github/workflows/release.yml

```diff
@@ -7,6 +7,23 @@ on:
 
 jobs:
 
+  trigger-docs-update:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.DOCS_UPDATE_TOKEN }}
+          script: |
+            github.rest.repos.createDispatchEvent({
```

Comment:
For the reference: https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/master/docs/repos/createDispatchEvent.md